spec/models/reservation_spec.rb

```diff
@@ -420,6 +421,85 @@
     end
   end
 
+  context 'with a reservation starting the next day that passes length limit' do
+    subject(:reservation) do
+      r = FactoryGirl.build(:valid_reservation)
+      r.equipment_model.max_checkout_length = 3
```

Comment:
You can replace these two lines with a call to `update_attribute` for conciseness.